.gitignore

```diff
@@ -1,6 +1,8 @@
 .zig-cache/
+zig-cache/
 zig-out/
 logs/
 settings.json
 gui_layout.json
-
+.cubyz/*
+!.cubyz
```

Comment:
it contains all the cubyz versions you download, not just the .gitignore. the .gitignore is there so that it is included in git, and i added that so that it doesnt send 80 billion files to github

Reply:
But why not just ignore the entire folder?